sdks/python/apache_beam/coders/coders.py

```diff
@@ -33,6 +36,11 @@
 from apache_beam.portability.api import beam_runner_api_pb2
 from apache_beam.utils import proto_utils
 
+try:
+  import cPickle as pickle
+except ImportError:
+  import pickle
```

Comment:
<!--new_thread; commit:21005b886b32241736b829e18f0aa386bbfbc13f; resolved:0-->
Please add comment that this is for Py2/3 compatibility, since in Python 3, cPickle was renamed pickle.